packages/hint-doctype/src/hint.ts

```diff
@@ -112,10 +189,22 @@ export default class implements IHint {
                 return;
             }
 
+            const fixes: {location: ProblemLocation; text: string}[] = [];
```

Comment:
nit: we use this type in a few places in the file. Consider defining it rather than doing this inline each time.